tests/integration/monitoring/test_deprecated_metric.py

```diff
@@ -0,0 +1,93 @@
+import time
+
+import requests as req
+from prometheus_client import CollectorRegistry, start_http_server
+
+from jina.serve.monitoring import _SummaryDeprecated
+
+from . import get_metric_values
```

Comment:
do not use this type of import, use absolute

Reply:
I am not a big fan on using absolute import in test